assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/mdsclient/DataCiteMdsConnection.java

```diff
@@ -30,6 +30,10 @@
     public static final String APPLICATION_XML_CHARSET_UTF_8 = "application/xml; charset=UTF-8";
     public static final String TEXT_PLAIN_CHARSET_UTF_8 = "text/plain;charset=UTF-8";
     public static final String LANDING_PAGE_BODY_FORMAT = "doi=%s\nurl=%s";
+    public static final String MISSING_DATACITE_XML_ARGUMENT =
```

Comment:
I'd write this in a more declarative way "The request did not contain UTF-8 encoded DataCite XML" (however, we are not actually checking either of these things, just that something is present) as this is system error logging, rather than an instruction that can be remedied immediately by a human. 

Ditto the following two constants